addons/print/models/print_printer.py

```diff
@@ -177,14 +191,56 @@ def spool(self, document, title=None, copies=1):
         return True
 
     def spool_report(self, docids, report_name, data=None, title=None, copies=1):
-        """Spool report to printer"""
+        """
+        Purpose: This function will render and spool the report to the printer that it
+        is called on, as long as the printer has a report_type attribute matching that of the desired
+        report.
+        Return: boolean
+        """
         # pylint: disable=too-many-arguments, too-many-locals
 
         if copies <= 0:
             _logger.info(_("Zero or fewer copies requested, nothing will be printed."))
             return True
 
-        # Identify reports
+        reports = self._fetch_reports_for_printing(report_name)
```

Comment:
Nit: This is personal in fact, i dont like the empty lines on every method. Maybe a comment if the name is not explanatory